### DISCUSSION TITLE
team members

### DISCUSSION THREAD

**Discussion by maggu2810** [openhab/openhab-core#3492] [Core Maintainers Discussions]
@kaikreuzer Is it correct that you are the only team member of "Core Maintainers"?

**Comment by kaikreuzer** [comment]
Indeed. I thought you were one as well (just added you now).
As the code in this repo is to 95% the previous ESH code, I wanted to start off with the both of us as the ESH committers/maintainers. But we should now quickly start the nomination/voting process for further maintainers (see [here](https://github.com/openhab/openhab-docs/pull/885/files#diff-b6568e6331ade1168a9e200698def2d9R31) for my suggested process, which is derived from the ESH processes, so I'd hope you are fine with them).

**Comment by maggu2810** [comment]
@kaikreuzer I used an emoji as a reply to your nomination. I assume you did not get a notification.

**Comment by kaikreuzer** [comment]
Correct, but that should be fine as a nominator should check the process at least once after one week.
